src/main/java/uk/gov/ons/census/action/model/entity/Case.java

```diff
@@ -95,4 +101,8 @@
 
   @Column(columnDefinition = "BOOLEAN DEFAULT false")
   private boolean handDelivery;
+
+  @Type(type = "jsonb")
+  @Column(columnDefinition = "jsonb")
+  private Map<String, String> metadata;
```

Comment:
Is it possible to use an Entity style class for this mapping instead of serialising all the data to strings and back ourselves?